Example/Core/Tests/GTMHTTPServer.m

```diff
@@ -0,0 +1,631 @@
+/* Copyright (c) 2010 Google Inc.
```

Comment:
Is this really from 2010? Remove the "(c)" to be consistent with Firebase copyrights and pass travis